### PR TITLE
[API-925] Update and expand auth instructions.

### PR DESCRIPTION
**Pull request by josh-hamacher-cobalt**
Updated the instructions to reflect the renaming of the `ApiKeyAuth` scheme to `ApiToken`.  Also expanded and cleaned up.

### CODE REVIEW

File: versions/v2/content/authentication.md [hunk 1]
```diff
@@ -17,17 +17,29 @@ You must replace <strong>YOUR-PERSONAL-API-TOKEN</strong> with your personal API
...
+- Scroll down to the `Response body` section.  Depending on how many orgs you belong to, you should
+  see one or more `resource` sections within the output.  Each of those sections should contain a
+  `token` field.  Select the org you want to experiment with and copy the `token` field value.  This
+  is your Org Token.
+- Return back to the _Authorize_ section.
+- Paste the Org Token into the `OrgToken` input and click the green _Authorize_ button below it.
+  The word `Authorized` should appear underneath the `OrgToken` input.
+- Close the dialog box again by clicking either _Close_ button.
+- All subsequent requests to `/assets`, `/findings`, `/pentests`, etc. will be scoped to your
+  personal API token and the organization selected.
```
Comment (mjang-cobalt) [inline versions/v2/content/authentication.md:42]: I'm suggesting a rewrite, based on the [Google Developer Style Guide](https://developers.google.com/style), and my personal tests of these steps. I'll be happy to explain each of my proposed changes if/as needed.

```suggestion
and built with the [OpenAPI specification](https://swagger.io/specification/).  To explore our API, complete the following steps:

- Create and copy your API token from your [Cobalt profile](https://app.cobalt.io/settings/api-token).
- Go to [Swagger](https://app.swaggerhub.com/apis/CobaltLab/cobalt-api/) and select the green
  _Authorize_ button on the right side of the screen.
- Paste your API token into the `ApiToken` text box.
- Select the green _Authorize_ button below the `ApiToken` input.  You should see `Authorized` 
   below the `ApiToken` label.
- Close the Available Authorizations dialog box.
- Search for the `Orgs` endpoint.
- Select the `GET /orgs` operation to expand it.
- Select the _Try it out_ button, and then select the _Execute_ button.
- Scroll down to the `Response body`. Identify and copy the `token` associated with your target
   organization (`org`). This is your Org Token.
- Scroll up and select the _Authorize_ button.
- Paste the Org Token into the `OrgToken` text box and select the green _Authorize_ button.
  You should see `Authorized` below the `OrgToken` label.
- Close the Available Authorizations dialog box.
- The OpenAPI (Swagger) tool will now include your API Token and Org Token with your next REST calls.
```

Comment (mjang-cobalt) [reply versions/v2/content/authentication.md:42]: Note: I'm not sure about the last step, as Swagger doesn't include the Org Token with my subsequent REST calls.

Comment (mjang-cobalt) [reply versions/v2/content/authentication.md:42]: After discussion with @josh-hamacher-cobalt , I wonder if we should even include steps associated with the "Org Token".

... even if it's there in the `Available Authorizations` dialog box

TLDR, our implementation of Swaggerhub doesn't include the "Org Token" in subsequent REST calls.

File: versions/v2/content/authentication.md [hunk 2]
```diff
@@ -17,29 +17,28 @@ You must replace <strong>YOUR-PERSONAL-API-TOKEN</strong> with your personal API
...
-- Close the dialog box again by clicking either _Close_ button.
-- All subsequent requests to `/assets`, `/findings`, `/pentests`, etc. will be scoped to your
-  personal API token and the organization selected.
+- Paste your API token into the `ApiToken` text box.
+- Select the green _Authorize_ button below the `ApiToken` input.  You should see `Authorized` below
+  the `ApiToken` label.
+- Close the Available Authorizations dialog box.
+
+Most (but not all) API calls are scoped to a specific organization.  This is done via the
+`X-Org-Token` header.  Any operations that require this will have that header as a required field.
```
Comment (mjang-cobalt) [inline versions/v2/content/authentication.md:32]: I like your addition. I add this suggestion to "simplify" the wording:

```suggestion
Most API calls are scoped to a specific organization, with the `X-Org-Token` header.
We include that header, as needed, in our API calls.
```

File: versions/v2/content/authentication.md [hunk 3]
```diff
@@ -17,29 +17,28 @@ You must replace <strong>YOUR-PERSONAL-API-TOKEN</strong> with your personal API
...
+
+To see which organizations you belong to:
+
+- Search for the `Orgs` endpoint.
+- Select the `GET /orgs` operation to expand it.
+- Select the _Try it out_ button, and then select the _Execute_ button (note that you do **not**
+  need to provide the `X-Org-Token` header here).
+- Scroll down to the `Response body`. Copy the `token` associated with your target organization
+  (`org`). This is your Org Token and should be provided in the `X-Org-Token` header when required.
```
Comment (mjang-cobalt) [inline versions/v2/content/authentication.md:41]:
```suggestion
  (`org`). This is your Org Token. Include it with the `X-Org-Token` header when required.
```

**Review by mjang-cobalt — APPROVED** [review]
Thanks so much for this, @josh-hamacher-cobalt

**Comment by josh-hamacher-cobalt** [comment]
No problem @mjang-cobalt , thank you for all of the great suggestions!

**Review by grayson-cobalt — APPROVED** [review]
